auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -153,6 +153,7 @@
             @remove-business="$emit('remove-business', $event)"
             @business-unavailable-error="$emit('business-unavailable-error', $event)"
```

Comment:
I was talking with Rodrigo, we should probably gun these and use the store instead

Later work though

Reply:
Agreed its not nice. 
I usually used event bus. I usually would use this: [mitt](https://github.com/developit/mitt).

In vue2 there is an option of emitting to the $root and then listening on it again (not 100% sure of syntax) but that is not supported in vue3. So would avoid that.

And I think adding every event to the store might get cumbersome, fast.
Tracking events with emitt is easy as you can just keep event names in enum/const.

Reply:
We can just make a specific store for entities, I think it would make it quite a bit easier - just used on the business dashboard

Reply:
I was looking at something like this perhaps:

https://runthatline.com/pinia-watch-state-getters-inside-vue-components/

Reply:
I still think it's important to use emit and props - but maybe we can shift a bit over to the store so it's less complicated?

Reply:
I usually use store for complex states, but for events, like I mentioned above, I use mitt library.
But yeah, if you want to swap it in the future, I wont mind :)
(I mean all the events are getting propagated, as the third level one is handling popups and all.